keyboards/inett_studio/sq80/info.json

```
@@ -0,0 +1,227 @@
+{
+    "keyboard_name": "SQUARE 80 %",
+    "manufacturer": "iNETT Studio",
+    "url": "https://geekhack.org/index.php?topic=100769.0",
+    "maintainer": "Wraul",
+    "usb": {
+        "vid": "0x694E",
+        "pid": "0x6060",
+        "device_version": "0.0.1"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "matrix_pins": {
+        "cols": [ "C7", "C6", "B6", "B3", "B4", "D7", "D6", "E6", "F0", "F1", "F4", "F5", "F7", "D5", "D3" ],
+        "rows": ["B0", "B1", "B2", "F6", "B5", "D4"]
+    },
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "D0",
+        "scroll_lock": "D1",
+        "num_lock": "D2",
+        "on_state": 0
+    },
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_tkl_ansi"
+    },
+    "community_layouts": ["tkl_ansi"],
+    "layouts": {
+        "LAYOUT_all": {
```

Need to use the "direct" name here, rather than the alias name.

```suggestion
        "LAYOUT_tkl_ansi": {
```

---

Good catch. Should be fixed now.